### ISSUE TITLE
Add accessibility support (inverted color scheme, font sizes)

### ISSUE THREAD

**Issue by ghost** [wireapp/wire#286]
A contact of mine is struggling to use the app due to a lack of accessibility features. She has to use OS-level magnification to make messages readable, and also has issues with the default color scheme.

"I get about five half-length lines on my 27" screen. Is there any chance of getting the text in an easier format? Black on white is kind of like staring into the headlights of a car for me."

Please add support for accessibility-friendly color schemes, such as white text on a black or dark grey background, as well as increasing font sizes within the application.

**Comment by jeroenpraat** [comment]
@ffflorian Can we please have a way to adjust the font size or at least be able to zoom the desktop app? For my eyes it's much to small.

**Comment by ffflorian** [comment]
@jeroenpraat in the [WebApp](https://app.wire.com) you can press <kbd>Ctrl</kbd>+<kbd>+</kbd> to zoom in. I will check if we can use that in the desktop app, too.

**Comment by iKarith** [comment]
You cannot zoom this way in the desktop app, sadly. (The ability to do so was explicitly disabled.)

**Comment by jeroenpraat** [comment]
> @jeroenpraat in the [WebApp](https://app.wire.com) you can press Ctrl++ to zoom in. I will check if we can use that in the desktop app, too.

Any update on this?

**Comment by jeroenpraat** [comment]
> @jeroenpraat in the [WebApp](https://app.wire.com) you can press Ctrl++ to zoom in. I will check if we can use that in the desktop app, too.

@ffflorian Any update on this?

**Comment by jeroenpraat** [comment]
This issue is related to https://github.com/wireapp/wire/issues/2892
